src/modules/learning-object-service/LearningObjectServiceController.ts

```diff
@@ -0,0 +1,34 @@
+import { Router } from "express";
+import { Controller } from "../../interfaces/Controller";
+import { ChangelogController } from "./changelogs/ChangelogController";
+import { CollectionController } from "./collections/CollectionController";
+import { FileAccessIdentitiesController } from "./fileAccessIdentities/FileAccessIdentitiesController";
+import { FileManagerController } from "./fileManager/FileManagerController";
+import { FileMetadataController } from "./fileMetadata/FileMetadataController";
+import { ObjectsController } from "./objects/ObjectsController";
+import { OutcomesController } from "./outcomes/OutcomesController";
+import { RevisionsController } from "./revisions/RevisionsController";
+import { SearchController } from "./search/SearchController";
+import { StatsController } from "./stats/StatsController";
+import { SubmissionsController } from "./submissions/SubmissionsController";
+
+export class LearningObjectServiceController implements Controller {
+    buildRouter(): Router {
+        const router = Router();
+
+        // Routes go here
+        router.use(new ChangelogController().buildRouter());
+        router.use(new CollectionController().buildRouter());
+        router.use(new FileAccessIdentitiesController().buildRouter());
```

Comment:
Is this just a second layer of controllers for the learning object service because it is so large?

Reply:
Yes.  LO service is broken up into smaller modules, unlike the other services.  It's our biggest service and really needs a separate controller for each sub module in that service.